src/home/WalletHome.tsx

```diff
@@ -56,6 +58,12 @@ function WalletHome() {
   const showNotificationCenter = getFeatureGate(StatsigFeatureGates.SHOW_NOTIFICATION_CENTER)
   const showNotificationSpotlight = showNotificationCenter && canShowNotificationSpotlight
 
+  const [hideBalances, setHideBalances] = useState(false)
+  const eyeballOnPress = () => {
+    setHideBalances(!hideBalances)
+    ValoraAnalytics.track(HomeEvents.hide_balances)
```

Comment:
should there be separate props for hide / show? Or at least include it as a prop?